src/Symbol.ts

```diff
@@ -61,11 +61,17 @@ export namespace Shim {
 			desc += String(postfix || '');
 			created[desc] = true;
 			name = '@@' + desc;
-			defineProperty(objPrototype, name, {
-				set: function (value: any) {
-					defineProperty(this, name, getValueDescriptor(value));
-				}
-			});
+
+			// FIXME: Temporary guard until the duplicate execution when testing can be
+			// pinned down.
+			if (!Object.getOwnPropertyDescriptor(objPrototype, name)) {
```

Comment:
When running `$ grunt test`, the loader executes certain factories twice, resulting in the well-known symbols being set twice. Once I track this down, I will remove this guard.